.github/ionic-issue-bot.yml

```diff
@@ -113,6 +114,8 @@ noReply:
   maxIssuesPerRun: 100
   label: "needs: reply"
   responseLabel: triage
+  exemptLabels:
+    - "needs: investigation"
```

Comment:
```suggestion
```

This isn't needed because the issue is only closed if it has `needs: reply` and we don't add that to investigative tickets